config.yaml

```diff
@@ -14,8 +14,8 @@ digital_outputs:
     gpio: 5
     mqtt_topic: "heatpump/output/i1"
     mqtt_topic_set: "heatpump/output/i1/set"
-    mqtt_state_on: "ON"
-    mqtt_state_off: "OFF"
+    mqtt_state_high: "ON"
+    mqtt_state_low: "OFF"
```

Comment:
This was renamed to be compatible with mqtt-io I assume? Just wanting to make sure I get the reason.

Reply:
mqtt-io supports an attribute on outputs called "inverse" that basically reverses the mapping of high to off and low to on.  I needed that for one of my switches.

Instead of adding an additional attribute and then making the calculation of what high and low should map to more complex, I figure it'd be simpler code to just directly configure the mapping.  However, I'm open to implementing it similar to mqtt-io by adding an inverse attribute if that makes more sense.

Reply:
Also, the implementation you had didn't actually use the yaml values; they were there, but never actually set in the Pin struct.  So there should be no concern about backwards compatibility since it was non-functional :)

Reply:
Oh - good catch. I indeed used the defaults only :) 

I agree, this is a pretty practical approach of just defining it that way - and you can just flip `ON/OFF` if inverse is required. 

Like it!